gnd/src/main/java/com/google/android/gnd/MainViewModel.java

```diff
@@ -67,11 +75,16 @@ public MainViewModel(
 
     // TODO: Move to background service.
     disposeOnClear(
-        projectRepository.getActiveProject().switchMapCompletable(this::syncFeatures).subscribe());
+        projectRepository
+            .getActiveProject()
+            .observeOn(schedulers.io())
```

Comment:
Do we need to pass the `subscribeOn` scheduler as well?

Reply:
The WIP best practices doc prescribes that we specify `subscribeOn()` in the method that defines the background operation (in this case `getActiveProject()`).